contracts/lending/ttoken/TToken_V1.sol

```diff
@@ -165,7 +166,9 @@ contract TToken_V1 is ITToken {
         uint256 onLoan = s().totalBorrowed - s().totalRepaid;
         uint256 supplied = totalUnderlyingSupply() + onLoan;
         if (supplied > 0) {
-            ratio_ = NumbersLib.ratioOf(onLoan + newLoanAmount, supplied);
+            ratio_ = SafeCast.toUint16(
+                NumbersLib.ratioOf(onLoan + newLoanAmount, supplied)
+            );
```

Comment:
I think given the comment from OZ, the `SafeCast.toUint16` should happen in the `NumbersLib.ratioOf` function. 

```solidity
        return num2 == 0 ? 0 : SafeCast.toUint16((num1 * ONE_HUNDRED_PERCENT) / num2);
```